src/web/src/components/CourseList.vue

```diff
@@ -32,46 +32,60 @@
         </b-col>
       </b-row>
     </div>
-
+    <!-- Start of Dynamic Scrolling Rendering To Account For Varying Course Data. > -->
     <hr />
     <div id="scroll-box" data-cy="course-list">
       <div v-if="filterCourses.length == 0" class="no-courses">
         Oops, no results!
       </div>
-      <recycle-scroller
-        class="scroller"
-        :items="filterCourses"
-        :item-size="100"
-        typeField="vscrl_type"
-        v-slot="{ item: course }"
-      >
-        <div class="course-listing" :class="{ 'bg-light': course.selected }">
-          <CourseListing
-            :course="course"
-            defaultAction="toggleCourse"
-            v-on="$listeners"
-            lazyLoadCollapse
-          >
-            <template #toggleCollapseButton="{ course }">
-              <button
-                v-show="
-                  course.corequisites ||
-                  course.prerequisites ||
-                  course.raw_precoreqs
-                "
-                class="btn"
-                @click.stop="courseInfoModalToggle(course)"
-                data-cy="course-info-button"
+      <template>
```

Comment:
Don't think you need a template here?

Reply:
I'm not sure, I tried to follow the Existing Code that was in place with Recycle Scroller. I'll try to see if removing this Template and its Corresponding one, changes anything.

Reply:
Hey Josh, 
I just tried it out, and was able to remove it. 
Let me know what you think and if all functionality is as expected. 
Thank you.